ts/main.ts

```diff
@@ -39,6 +57,87 @@ $entryForm.addEventListener('submit', (event: Event) => {
   data.nextEntryId++;
   data.entries.unshift(entryValues);
   writeData();
+  const $newEntry = renderEntry(entryValues);
+  $allEntriesUl.prepend($newEntry);
   $entryImg.setAttribute('src', '/images/placeholder-image-square.jpg');
   $entryForm.reset();
+  viewSwap('entries');
+  if (data.entries.length > 0) {
+    toggleNoEntries();
+  }
+});
```

Comment:
By making the change to the `toggleNoEntries` function, you also won't need to have the same condition here. It'll allow you to only write the condition once but it will affect different both areas. (Plus you may be able to reuse it when you get to the 4th feature of deleting entries)